CITATION

```diff
@@ -1,8 +1,15 @@
 If you use Gammapy for work/research presented in a publication (whether
 directly, or as a dependency to another package), we recommend and encourage
-the following citation scheme:
-- refer to the Gammapy paper as general citation: A&A, 678, A157 (2023) - the bibtex is given below
-- AND cite the version of Gammapy used with the DOI 10.5281/zenodo.4701488
+the following citation scheme (see also https://gammapy.org/acknowledging.html):
+- refer to the Gammapy paper as general citation: A&A, 678, A157 (2023) - the bibtex is given below,
+- AND cite the used version of Gammapy (e.g. see https://doi.org/10.5281/zenodo.14760974 for the v1.3),
+- AND add the acknowledgment given below.
```

Comment:
Do we really want to recommend three different ways to cite and acknowledge gammapy?

Reply:
The proposal is about the 3, not 2 as before... This is the point of the discussion here: do we add the third or not?

Reply:
I would not add the third. Instead encourage people to cite other dependencies

Reply:
I think I do not understand your point, as it seems to say that this PR is not needed...
But in the dev call, we mentioned the wish to acknowledge the teams of our dependencies...
Are we going backward?